pkg/hydrate/flatten.go

```diff
@@ -16,7 +16,7 @@ package hydrate
 
 import (
 	corev1 "k8s.io/api/core/v1"
-	v1 "kpt.dev/configsync/pkg/api/configmanagement/v1"
```

Comment:
The function in this file is unused (aside from unit test) and can be removed

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1016